content/en/agent/iot/_index.md

```diff
@@ -0,0 +1,159 @@
+---
+title: IoT Agent
+kind: documentation
+beta: true
+further_reading:
+  - link: "/getting_started/agent/"
+    tag: "Documentation"
+    text: "Getting Started with the Agent"
+---
+
+## Overview
+
+The Datadog IoT Agent is a version of the Agent optimized for monitoring IoT devices and embedded applications. Customers use the IoT Agent to monitor a wide variety of devices from digital displays to security devices running image detection algorithms.
+
+<div class="alert alert-warning">
+The IoT Agent is currently available for private beta. Request access by completing the <a href="https://www.datadoghq.com/">form</a>.
+</div>
+
+## Capabilities
+
+The IoT Agent includes the following system checks. Configuration is identical to the full Agent.
+
+- [System][1] (includes CPU, IO, load, memory, swap, and uptime)
+- [Disk][2]
+- [Network][3]
+- [Systemd][4]
+- [NTP][5]
+
+The IoT Agent also supports:
+
+- Custom metric collection using an embedded [DogStatsD][6] server
+- Log collection using [tailing files][7], [TCP/UDP][8], and [journald][9]
+
+The IoT Agent does not include the Python interpreter and other integrations pre-packaged with the full Agent. It also doesn’t support tracing for APM, live process monitoring, or network performance monitoring.
+
+## Setup
+
+### Requirements
+
+The IoT Agent is available on Debian and RPM-based systems for both 64-bit x86 and Arm v8 devices.
```

Comment:
```suggestion
The IoT Agent is available as DEB and RPM packages for Linux devices running on x64, arm64 (ARMv8) and ARMv7 architectures.
```
- Explicitly mention Linux is the only supported OS.
- IoT Agent 7.21 will be available for ARMv7 already.